src/main/java/io/nats/client/Options.java

```diff
@@ -678,6 +678,7 @@ public class Options {
 
     private final List<java.util.function.Consumer<HttpRequest>> httpRequestInterceptors;
     private final Proxy proxy;
+    private final boolean enableFastFallback;

```

Comment:
This property needs an environment/property variable. The goal is that every single option can be set via a property. Look at PROP_TLS_FIRST for an example